charts/mlrun-ce/values.yaml

```diff
@@ -405,3 +405,63 @@ kube-prometheus-stack:
     fullnameOverride: state-metrics
   prometheus-node-exporter:
     fullnameOverride: node-exporter
+
+tdengine:
+  enabled: true
```

Comment:
Add to [admin_installation_values.yaml](https://github.com/mlrun/ce/blob/development/charts/mlrun-ce/admin_installation_values.yaml) :
```
tdengine:
  enabled: false
```

Reply:
Done